Havit.Blazor.Components.Web.Bootstrap/Forms/HxRadioButtonListBase.cs

```diff
@@ -142,13 +149,14 @@ protected void BuildRenderInput_RenderRadioItem(RenderTreeBuilder builder, int i
 
 			string inputId = GroupName + "_" + index.ToString();
 
-			builder.OpenElement(100, "div");
-
-			// TODO CoreCssClass
-			builder.AddAttribute(101, "class", CssClassHelper.Combine("form-check", Inline ? "form-check-inline" : null, ItemCssClassImpl, ItemCssClassSelectorImpl?.Invoke(item)));
-
+			if (inputAsToggleEffective != Bootstrap.InputAsToggle.Toggle)
+			{
+				builder.OpenElement(100, "div");
+				// TODO CoreCssClass
+				builder.AddAttribute(101, "class", CssClassHelper.Combine("form-check", Inline ? "form-check-inline" : null, ItemCssClassImpl, ItemCssClassSelectorImpl?.Invoke(item)));
+			}
```

Comment:
Careful, this ignores `ItemCssClass` and `ItemCssClassSelector` for toggle buttons.  
We need to decide whether we can make these work somehow, or if the combination with toggle buttons will be unsupported (in which case, we should probably throw an exception in `OnParametersSet`).